### ISSUE TITLE
Button Component does not announce Checked/Unchecked State

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30959]
## Description
Does not announce "checked"/"not checked" on focus of Button Component.

This issue is covered in-depth by parent issue #30843, please check there for more details.

## React Native version:
0.63

**Comment by anaskhraza** [comment]
Hi @amarlette
I'd like to contribute to this issue. Can you assign it to me, please?

**Comment by amarlette** [comment]
@anaskhraza thank you for taking this one! Remember to tag this issue in your pull request. I look forward to getting your PR on the board

**Comment by anaskhraza** [comment]
@amarlette I have investigated the issue, I think this PR #31001 has already resolved this issue. Can you please verify my findings?

**Comment by amarlette** [comment]
@anaskhraza I believe that the fix will be very similar but don't know if that PR fixes this issue. I will get back to you.

**Comment by huzaifaaak** [comment]
@amarlette I tested this and it:

- Announces **checked** when checked when `accessibilityState={{checked: true}}`
- Announces **not checked** when `accessibilityState={{checked: false}}`

**Comment by amarlette** [comment]
Ok thank you @huzaifaaak for the help! I will link that PR and close out this ticket.

**Comment by amarlette** [comment]
Closed by #31001

**Comment by huzaifaaak** [comment]
@kacieb @amarlette should I write test for this ?
